app.js

```diff
@@ -1,25 +1,27 @@
 // define all the variables and add 'div' or 'span' for clarity
+import { getComputerChoice } from './functions.js';
+
 let userScore = 0;
 let computerScore = 0;
 let drawScore = 0;
 const userScore_span = document.getElementById('user-score');
 const computerScore_span = document.getElementById('computer-score');
 const drawScore_span = document.getElementById('draw-score');
 const result_div = document.querySelector('.result > h1');
-const scoreBoard_div = document.querySelector('score-board');
-const rock_div = document.getElementById('r');
-const paper_div = document.getElementById('p');
-const scissors_div = document.getElementById('s');
+// const scoreBoard_div = document.querySelector('score-board');
+const rock_div = document.getElementById('rock');
+const paper_div = document.getElementById('paper');
+const scissors_div = document.getElementById('scissors');
```

Comment:
I really liked how you refactored this to have actual names other than 'r' 'p' 's', this makes the code way more readable and understandable!